### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN010.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#56] [General]
# https://blog.3place.cn/3place/lmyn/LMYN010.html

https://blog.3place.cn/3place/lmyn/LMYN010.html

**Comment by picklecai** [comment]
他不懂得女人的幸福，
也不懂得感官的享乐。 

要利用感官的享乐，
它们扰乱人的思想，
我们把他引进城来，
要迫使他尝上一尝。

堂堂宰相，不过是梅菲斯特附体。

**Comment by picklecai** [comment]
所有这些女子走了以后，
毗槃吒迦的儿子婆罗门，
心里面凄凄惶惶地不安，
他完完全全陷入了苦闷。

作为经典作品，没有胡说八道，没有一上来就引诱完成。今天的爽文，真不如人类孩童时代。那时虽然稚嫩，却尊重现实。

**Comment by picklecai** [comment]
这高贵尊严的婆罗门
被她们引诱走了以后，
天老爷立刻就下了雨，
全世界都精神抖擞。 

有人知道为啥能下雨吗？

**Comment by picklecai** [comment]
国王听了鸯输曼的话，
这话听起来实在可怕。
按照规定，按照仪节，
他就把这祭祀举行罢。 （1.40.24）
国王回到了自己城里，
他卓越而又渴望致祭；
怎样才能使恒河下凡，
他却实在想不出主意。 （1.40.25）

为什么国王听了这话后想要恒河下凡？他要祭祀恒河吗？
